src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/link/_create.py

```diff
@@ -9,7 +9,23 @@
 # flake8: noqa
 
 from azure.cli.core.aaz import *
-
+from azure.cli.core.breaking_change import register_output_breaking_change, register_argument_deprecate
+
+register_output_breaking_change('sql mi link create',
+                                description='Deprecated output properties: targetDatabase, '
+                                            'primaryAvailabilityGroupName, secondaryAvailabilityGroupName'
+                                            'sourceEndpoint, sourceReplicaId, targetReplicaId, '
+                                            'linkState, lastHardenedLsn.'
+                                            '\nNew output properties: databases, partnerAvailabilityGroupName,'
+                                            ' instanceAvailabilityGroupName, partnerEndpoint,'
+                                            ' distributedAvailabilityGroupName, instanceLinkRole,'
+                                            ' partnerLinkRole, failoverMode, seedingMode.',
+                                doc_link='aka.ms/mi-link-rest-api-create-or-update')
+
+register_argument_deprecate('sql mi link create', '--target-database', '--databases')
+register_argument_deprecate('sql mi link create', '--primary-availability-group-name', '--partner-availability-group-name')
+register_argument_deprecate('sql mi link create', '--secondary-availability-group-name', '--instance-availability-group-name')
+register_argument_deprecate('sql mi link create', '--source-endpoint', '--partner-endpoint')
```

Comment:
If I remember correctly, all these `_create`, `_list`, `_show`, `_update` files are auto-generated, right? To avoid manually update auto-generated files, you can create a new `_breaking_change` file and register all upcoming breaking changes

Reply:
If they are auto-generated, are then my changes in these PR useless: https://github.com/Azure/azure-cli/pull/30031 ?

How can I update these commands otherwise. I didn't now that they were autogenerated since the file path does not suggest that. What is then the way of updating these commands to use new SDK/API ?

Reply:
I have added _breaking_change.py for this PR.